Podfile

```diff
@@ -2,7 +2,7 @@ source 'https://github.com/CocoaPods/Specs.git'
 
 use_frameworks!
 
-pod 'IQKeyboardManagerSwift', '~>3.3.7'
+pod 'IQKeyboardManagerSwift'
```

Comment:
Sure - I figured the version is enforced in the .lock, and we update this project frequently enough that it'd be fine if a `pod update` grabbed the latest. But, also no harm in being a bit more explicit.